sorting/bubblesort.py

```diff
@@ -0,0 +1,34 @@
+"""
+Bubble Sort adalah Sorting algorithm yang cara kerjanya adalah dengan
+membandingkan 2 elemen array lalu menggeser kedua elemen tersebut
+sesuai dengan urutan terus-menerus sampai akhir array
+"""
+
+
```

Comment:
LGTM, tinggal tambahkan aja
```suggestion
from __future__ import annotations
```

https://mypy.readthedocs.io/en/latest/runtime_troubles.html